base/ltmiscen.dtx

```diff
@@ -1188,11 +1188,27 @@
 %  print as the \TeX{}book's space character instead of as blank spaces.
 %
 %  \begin{macro}{\@vobeyspaces}
+%  \begin{macro}{\@vobeytabs}
+% \changes{v1.2b}{2023/06/14}{Macro added}
 %    \begin{macrocode}
 {\catcode`\ =\active%
 \gdef\@vobeyspaces{\catcode`\ \active\let \@xobeysp}}
+%</2ekernel>
+%<latexrelease>\IncludeInRelease{2023/11/01}%
+%<latexrelease>                 {\@vobeytabs}{Obeyed tabs}%
+%<*2ekernel|latexrelease>
+{\catcode`\^^I=\active
+\gdef\@vobeytabs{\catcode`\^^I\active\let^^I\@xobeytab}}
```

Comment:
are we sure that it is better to have a separate command or to stick the TAB handling into \@vobeyspaces? I'm just wondering, but there are several packages out there that mimic verbatim setup and maybe they would not have to change and nevertheless pick up the  ^Î handling this way. Just as thought ... not thought through

Reply:
I can see this both ways. I agree that if you want to say 'basically tabs are spaces' then yes, it makes sense not to have a separate macro. But the moment you handle tabs there's the possibility of diverging from space handling, and it was that I was thinking of: in particular, people could allow a tab to convert to multiple spaces in verb material.

Reply:
Ah, you mean include `\@vobeytabs` in the def. for `\@vobeyspaces`?

Reply:
I didn't mean this originally, but yes, your argument about  being able to adjust it is valid, so I now thought moving it in there gives you the best of both worlds